config/locales/mailers/order_copy_cards_mailer.en.yml

```diff
@@ -4,26 +4,26 @@ en:
       send_order_completed_email:
         subject: "We’re printing your waste carriers registration cards"
         dear: "Dear %{full_name}"
-        heading: "We’re printing your waste carriers registration cards. They should arrive within 10 working days."
+        heading: "We’re printing your waste carriers registration cards. They should arrive within 10 working days"
```

Comment:
I have spoken with @andrewhick and confirmed this. If we are trying to ensure the pluralisation is correct for copy cards, then this should include the subject and heading in the email as well, not just the back office UI.

And as this PR is focused on fixing these issues, I thought it would make sense to make those changes here rather than as subsequent one. However, if you don't agree I'm happy to approve. We'll just need another PR for handling pluralisation in the email content.

Reply:
Since doing that requires to add a `quantity ` field to the OrderItem object, I think I will open a different PR :)